contracts/AuctionBase.sol

```diff
@@ -110,10 +124,11 @@ contract AuctionBase is Pausable {
       tokenId: _tokenId,
       seller: msg.sender,
       bidIncrement: uint128(_bidIncrement),
-      duration: uint64(_duration),
-      startedAt: uint64(now),
+      duration: durationBlockCount,
+      startedAt: now,
```

Comment:
Things to consider: `use of "now": "now" does not mean current time. Now is an alias for block.timestamp. Block.timestamp can be influenced by miners to a certain degree, be careful.` Maybe we don't keep this?

Reply:
Yeah, aware of this. I think having this as an approximation for now is fine. Alternatively, we can pass a timestamp from the client and store that. But will keep it for now.